### ISSUE TITLE
ERROR! WEBP_LIBRARIES

### ISSUE THREAD

**Issue by Neotronik** [aseprite/aseprite#3831]
Error while compiling

I inserted this code
```
cd aseprite
mkdir build
cd build
export CC=clang
export CXX=clang++
cmake \
  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
  -DCMAKE_CXX_FLAGS:STRING=-stdlib=libc++ \
  -DCMAKE_EXE_LINKER_FLAGS:STRING=-stdlib=libc++ \
  -DLAF_BACKEND=skia \
  -DSKIA_DIR=$HOME/deps/skia \
  -DSKIA_LIBRARY_DIR=$HOME/deps/skia/out/Release-x64 \
  -DSKIA_LIBRARY=$HOME/deps/skia/out/Release-x64/libskia.a \
  -G Ninja \
  ..
ninja aseprite

```
And got this error:
```

-- aseprite libwebp: WEBP_LIBRARIES-NOTFOUND
-- Looking for include file xcb/xcb.h
-- Looking for include file xcb/xcb.h - found
-- Looking for include file png.h
-- Looking for include file png.h - found
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
WEBP_LIBRARIES
    linked by target "app-lib" in directory /home/..../aseprite/src/app

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
ninja: error: loading 'build.ninja': No such file or directory
```
Does anyone know how to fix this?

* Aseprite version: 1.2.40
* System: Linux Ubuntu 22.04

**Comment by Neotronik** [comment]
> Hi @Neotronik, I highly recommend you to try to compile the latest v1.3-rc2 or the main branch with the latest Skia version we are distributing. You can also check #3628 for some possible solutions.

Thanks for the answer! I'll try.

**Comment by Neotronik** [comment]
> Hi @Neotronik, I highly recommend you to try to compile the latest v1.3-rc2 or the main branch with the latest Skia version we are distributing. You can also check #3628 for some possible solutions.

It didn't work. I installed the latest version Aseprite and the latest version Skia, but the error is the same. Even the library `libwebp-dev` didn't help. I also tried the tips from #3628.

**Comment by orende** [comment]
Can confirm that this same error is affecting me as well.

**Comment by PeterPedro** [comment]
yeah me too, changed the path in CMakeLists.txt to "/third_party/libwebp/src" but it didn't work

**Comment by dacap** [comment]
Hi @orende what specific version/commit are you trying? we highly recommend to try the latest v1.3-rc4 or the main branch.

**Comment by dacap** [comment]
@Neotronik just in case, after this error appears, I'd recommend to delete the whole build folder and start from scratch. Other last resort option is just disabling the webp support (changing `ENABLE_WEBP:BOOL=OFF` in the CMakeCache.txt)

**Comment by Jessicacasino007** [comment]
Ma a effort for speaker french

Le ven. 26 mai 2023 10 h 04, David Capello ***@***.***> a
écrit :

**Comment by flutterdro** [comment]
I head the same issue but it turned out that it is just me dumdum. 
In my case I just unziped contents of the Skia-macOS-Release-arm64.zip into the home/deps folder. After that I simply copied cmake command and ran it with this option -DSKIA_LIBRARY=$HOME/deps/skia/out/Release-arm64/libskia.a which was the root of the issue since my path was actually home/deps/Skia-macOS-Release-arm64/out/Release-arm64/libskia.a. After I modified the paths all the issues disappeared. I hope it would help at least someone
